exporter/signalfxexporter/eventclient.go

```
@@ -94,7 +93,10 @@ func (s *sfxEventClient) pushLogsData(ctx context.Context, ld plog.Logs) (int, e
 	}
 
 	defer func() {
-		io.Copy(ioutil.Discard, resp.Body)
+		_, err := io.Copy(ioutil.Discard, resp.Body)
+		if err != nil {
+			s.logger.Warn(err.Error())
+		}
```

ignore the error for the moment, don't log it.